### ISSUE TITLE
Enable directory search by sql

### ISSUE THREAD

**Issue by Rohithzr** [kamax-io/mxisd#47]
I am trying to activate directory search as documented in [directory-users.md](https://github.com/kamax-io/mxisd/blob/40705b5d472a64d646b16b6e6d171c42b4848a27/docs/features/directory-users.md#sql)

I see that the query passes one param that should be i believe the seach query? but when I see in the database :
```
Query:
select * from user_directory;
Response:
userId: @username:idigid.com
displayName: NULL
```
now this user just signed up so no display name is present but is this the right behavior?  shouldn't i be able to search the user with "username" and not needing to write "@username:idigid.com"

**Comment by maxidor** [comment]
The SQL module documentation is not as good as I want it to be, and I don't use it myself.
I'll get back to you on this after I've confirmed that everything is working as expected.

**Comment by Rohithzr** [comment]
it would be great if i can pass the 'localpart' multiple times something like
```
type: 'localpart,localpart'
value: 'SELECT idColumn, displayNameColumn FROM table WHERE displayNameColumn LIKE ? OR idColumn LIKE %?%'
```

**Comment by maxidor** [comment]
This is a subset of #56

**Comment by maxidor** [comment]
@Rohithzr After looking into this, there doesn't seem to be an issue. I think the lack of documentation is the real problem, so we'll fix that as part of #56.
About the [multiple parameters in the query](https://github.com/kamax-io/mxisd/issues/47#issuecomment-353188355), mxisd already replaces every `?` in the query by the search value. There is no notion of "type": the string sent by the client is used.

I'll close this. Please re-open once #56 is completed if you are still having problems.
